BOJ/4673/hikang.py

```diff
@@ -0,0 +1,16 @@
+# 생성자 구현
+def d(n):
+  return n + sum(map(int,str(n)))
+
+sNum = set()
+# 생성자가 있는 경우에 sNum 집합에 추가
+for i in range(1,10001):
+  a = d(i)
+  if a < 10001:
+    sNum.add(a)
+
+# sNum 집합에 없으면 셀프넘버라는 의미
+# 범위 전체에서 셀프넘버인 경우 출력
+for i in range(1,10001):
+  if i not in sNum:
+    print(i)
```

Comment:
```suggestion
for i in list(set(range(1,10001) - sNum):
  print(i)
```
생성자가 있는 집합이 `sNum` 세트니까 차집합 연산을 하면 훨씬 보기 이쁘고 효율적으로 보일 수 있지

`list`로 자료형 바꾼 이유는 순서대로 출력해야해서

Reply:
오오 차집합 굿굿! 더 간결해보인다 떙큐~~~